apps/prairielearn/src/pages/instructorAssessmentQuestions/instructorAssessmentQuestions.ts

```diff
@@ -46,6 +113,38 @@ router.post(
         authn_user_id: res.locals.authn_user.user_id,
       });
       res.redirect(req.originalUrl);
+    } else if (req.body.__action === 'edit_assessment_questions') {
+      const assessmentPath = path.join(
+        res.locals.course.path,
+        'courseInstances',
+        res.locals.course_instance.short_name,
+        'assessments',
+        res.locals.assessment.tid,
+        'infoAssessment.json',
+      );
+      const paths = getPaths(req, res);
+      const assessmentInfo = JSON.parse(await fs.readFile(assessmentPath, 'utf8'));
+      const origHash = req.body.__orig_hash;
+      assessmentInfo.zones = JSON.parse(req.body.zones);
```

Comment:
Note to self: before this lands, ensure this is using the new formatting/key-order goodness from the access rules editor PR.